wpilibj/src/shared/java/edu/wpi/first/wpilibj/smartdashboard/SmartDashboard.java

```diff
@@ -36,7 +37,7 @@
    * A table linking tables in the SmartDashboard to the {@link Sendable} objects they
    * came from.
    */
-  private static final Hashtable tablesToData = new Hashtable();
+  private static final Map<Sendable, String> tablesToData = new HashMap<>();
```

Comment:
@SamCarlberg since this one isn't related to Command-based at all, this definitely needs to be looked at. Teams will definitely use SmartDashboard from multiple threads.